properties/metadata/modified.md

```diff
@@ -0,0 +1,14 @@
+# modified
```

Comment:
Would it be possible to rename this property and file to be `modified-timestamp` so that it's more clear that it would be associated with a timestamp (vs. modified which sounds like a boolean value)?

Reply:
Yes.  done.